src/confluent_kafka/experimental/aio/producer/_kafka_batch_executor.py

```diff
@@ -106,10 +108,24 @@ def _produce_batch_and_poll() -> int:
         loop = asyncio.get_running_loop()
         return await loop.run_in_executor(self._executor, _produce_batch_and_poll)
 
-    def _handle_partial_failures(
-        self,
-        batch_messages: List[Dict[str, Any]]
-    ) -> None:
+    async def flush_librdkafka_queue(self, timeout=-1):
```

Comment:
I think since you picked up master changes you now have some more methods to type -- I'll approve PR and we can do a second pass for these separately as part of putting code checks on typing.